docs/npkillrc.md

```
@@ -0,0 +1,206 @@
+# Config File - npkillrc
+
+You can customize the behavior of npkill through the config file (`.npkillrc` by default).
+
+## Table of Contents
+
+- [Location](#location)
+- [Example](#example)
+- [Options](#options)
+  - [rootDir](#rootDir)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid
Expected: #rootdir; Actual: #rootDir

(MD051, link-fragments)